eng/common/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -10,8 +10,10 @@ parameters:
 steps:
 - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 - pwsh: |
-    Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://aka.ms/downloadazcopy-v10-windows" -OutFile "azcopy.zip" | Wait-Process;
-    Expand-Archive -Path "azcopy.zip" -DestinationPath "$(Build.BinariesDirectory)/azcopy/"
+    if (!(Test-Path '$(Build.BinariesDirectory)/azcopy/azcopy_windows_amd64_*/azcopy.exe')) {
```

Comment:
These changes will need a rebase otherwise merge will be blocked since they are in eng/common.

Reply:
I think it all merged cleanly, but please let me know if I should follow up!